base/utils/tests/testthat/test.distn.stats.R

```diff
@@ -13,7 +13,7 @@ test_that("distn.stats works for different distributions",{
 })
 
 test_that("distn.table.stats works for different distributions",{
-  data(prior.distns)
+  load("data/prior.distns.RData")
```

Comment:
Wait, aren't we going to move this to the tests/data directory?  Or did we decide to not lazy load?

Reply:
From what @dlebauer and @infotroph  were saying I believe the plan is to put them in the `tests` directory within each package directory: `tests/testthat/data`.  So for this they're in `base/utils/tests/testthat/data`. I believe this avoids lazy loading the data since it's in the `tests` directory and allows each package to stand alone and not have to depend on files from another package.

Reply:
@tonygardella Great, I agree with that!

Reply:
@tonygardella I can't tell is load(data/prior.distns.RData will try to load from tests/data or data/?  DO you know which it is trying to load from?

Reply:
sorry, I meant tests/testthat/data.....

Reply:
It would have to be the "tests/testthat/data" because they're no longer in the "data" directory of the package.

Reply:
OK, so you will update that so it is grabbing from the correct place in the test?  And then we can approve...

Reply:
Hmmm..Oh I see what you mean. I thought that the test scripts are run from the directory they are in. So since they're in "tests/testthat" I thought the priors could be loaded from just adding the relative path of "data/prior.distns.RData". I assume this works because `make test` works and that runs all the tests.

Reply:
I'm essentially following what @infotroph said in his third point, but instead of loading priors in the meta.analysis package from utils I put it over too as @dlebauer suggested.

Reply:
Ahhh....OK then.  I wasn't thinking about relative paths. 

Have you run through to make sure you are now getting the correct priors/posteriors in a PEcAN workflow to test the change?